runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -267,6 +273,19 @@ public void onPlayerLootReceived(final PlayerLootReceived playerLootReceived)
 		}
 	}
 
+
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
+	{
+		if (!shouldTakeGauntletScreenshot || event.getContainerId() != InventoryID.INVENTORY.getId())
+		{
+			return;
+		}
+
+		shouldTakeGauntletScreenshot = false;
+		takeScreenshot("Gauntlet Loot", "Gauntlet Loot");
```

Comment:
I'd put this in the `"Boss Kills"` folder with other similar loot. (think Chambers of Xeric, Theatre of Blood)

Also, this currently doesn't record the actual gauntlet completion kill count. We should get this from the completion message like is done for other similar activities. (this could also use different filenames for regular vs. corrupted gauntlet)

Reply:
Happy to move to `Boss Kills` putting it in the same vein as raids. But I have a question about using KC in the filename


_"Completion of The Gauntlet will reward players with various loot, including crystal armour seeds for crystal armour, the Blade of Saeldor, and crystal shards to charge the previous items. Failing to finish, either by death or leaving early, will give a small token reward based upon how well the player performed, but won't grant any chance of unique items."_ - OSRS Wiki

A player can die in a Gauntlet run and receive loot. In these scenarios, the KC is neither incremented nor referenced

Thinking of 3 potential options

- We use KC in the filename and ignore screenshotting bad Gauntlet runs (my preference, though there's potential for confusion when loot is in loot tracker but not screenshotted)
- We use KC in the filename and set a special name or timestamp only for bad Gauntlet runs (eh)
- We don't use KC in the filename and use timestamp only for Gauntlet runs (similar to pet)

Reply:
If you can get gauntlet loot without kc, then this is probably fine. Maybe just name the folder `Gauntlet`?

Reply:
Done 337c0677acde349f8e625dca7ad16b15a8d872bf